CODEOWNERS

```diff
@@ -18,110 +20,111 @@
 # @cilium/policy        Policy behaviour
 # @cilium/proxy         L7 proxy, Envoy
 # @cilium/vendor        Vendoring, dependency management
-#
-# Specific code owners:
-# @eloycoto             Debian packaging
-# @aanm @ianvernon      Docker packaging
-# @scanf @borkmann      BPF documentation
 
 # The following filepaths should be sorted so that more specific paths occur
 # after the less specific paths, otherwise the ownership for the specific paths
 # is not properly picked up in Github.
 * @cilium/janitors
-api/ @cilium/api
-bpf/ @cilium/bpf
-bugtool/cmd/ @cilium/cli
-cilium/ @cilium/cli
-cilium/cmd/preflight_k8s_valid_cnp.go @cilium/kubernetes
-cilium-health/cmd/ @cilium/cli
-cilium-health/launch/ @cilium/health
-contrib/packaging/deb/ @eloycoto
-contrib/packaging/docker/ @aanm @ianvernon
-contrib/vagrant @cilium/ci
-daemon/ @cilium/agent
-daemon/cmd/datapath.* @cilium/bpf
-daemon/cmd/endpoint.* @cilium/endpoint
-daemon/cmd/health.* @cilium/health
-daemon/cmd/hubble.go @cilium/hubble
-daemon/cmd/ipcache.* @cilium/ipcache
-daemon/cmd/loadbalancer.* @cilium/loadbalancer
-daemon/cmd/metrics.* @cilium/metrics
-daemon/cmd/policy.* @cilium/policy
-daemon/cmd/prefilter.go @cilium/bpf
-daemon/cmd/proxy.go @cilium/proxy
-daemon/cmd/state.go @cilium/endpoint
-daemon/cmd/sysctl_linux.go @cilium/bpf
-Documentation/ @cilium/docs
-Documentation/bpf.rst @scanf @borkmann
-Documentation/contributing/ @cilium/contributing
-envoy/ @cilium/proxy
-examples/ @cilium/docs
-examples/kubernetes/ @cilium/kubernetes
-examples/minikube/ @cilium/kubernetes
+/api/ @cilium/api
+/bpf/ @cilium/bpf
+Makefile @cilium/build
+Dockerfile @cilium/build
```

Comment:
Do we need to wildcard this to catch the builder/runtime images?

```
$ find . -name "Docker*"
./Dockerfile.builder
./test/startup-script/Dockerfile
./Dockerfile
./Documentation/Dockerfile
./.git/logs/refs/remotes/eloycoto/DockerLogsNames
./.git/logs/refs/remotes/eloycoto/DockerLogs
./vendor/github.com/pelletier/go-toml/Dockerfile
./vendor/github.com/hpcloud/tail/Dockerfile
./vendor/github.com/envoyproxy/protoc-gen-validate/Dockerfile
./vendor/golang.org/x/net/http2/Dockerfile
./examples/kubernetes-grpc/Dockerfile
./contrib/packaging/rpm/Dockerfile
./contrib/packaging/docker/Dockerfile.runtime
./contrib/packaging/deb/Dockerfile
```

Reply:
FYI, We'll already catch `Dockerfile.runtime` in `@cilium/build` because it's under `contrib/packaging/`.